src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -840,17 +833,29 @@ def load_arguments(self, _):
             "enable_advanced_network_observability",
             action="store_true",
             is_preview=True,
+            deprecate_info=c.deprecate(
+                target="--enable-advanced-network-observability",
```

Comment:
would be better to add declarations for newly added parameters disable_acns_observability/disable_acns_security

Reply:
i can update but i thought the `--enable-acns` command would be more similar to the now depircated `--enable-advanced-network-observability` as they both enable the `advancednetworking.observability.enabled` property